client/src/components/Homepage/Rules.jsx

```diff
@@ -1,10 +1,81 @@
-import React from 'react';
+import React, { useState } from 'react';
+import('./Rules.css');
+import { Typography, Button, Card, Grid, Container } from '@material-ui/core';
+import { makeStyles, withStyles } from '@material-ui/core/styles';
+import { useHistory } from 'react-router-dom';
+import SidebarDrawer from './Drawer';
+
+const styles = {
+  root: {
+    background: 'linear-gradient(45deg, green 30%, orange 90%)',
+    border: 0,
+    borderRadius: 3,
+    boxShadow: '0 3px 5px 2px rgba(255, 105, 135, .3)',
+    color: 'white',
+    height: 48,
+    padding: '0 30px',
+  },
+};
 
 const Rules = () => {
   const text = 'rules';
+  const history = useHistory();
+  const classes = styles;
+  const [showDrawer, setShowDrawer] = useState();
+  const [accepted, setAccepted] = useState(false);
+  console.log(accepted);
+
   return (
-    <div>{text}</div>
+    <div>
+      <SidebarDrawer />
+      <Grid container direction="column" alignItems="center">
+        <Typography variant="h1">Rules</Typography>
+        <Grid
+          container
+          direction="column"
+          alignItems="center"
+          className="mainContainer"
+        >
+          <Grid item className="gridItem">
+            Each user will be given 1 resource at the beginning of their turn.
```

Comment:
rather than hard coding this on the frontend, we should save this to the database and retrieve it from a call, alternatively, you could save it in a local json file and get the information from that